.github/workflows/ci.yml

```diff
@@ -61,3 +62,4 @@ jobs:
         with:
           working-directory: ./websites/visualization
           comment-strategy: 'always'
+          create-issue: false
```

Comment:
I'll make sure no issue is created. https://github.com/toss/suspensive/issues/1128

Reply:
But if we need to diff default branch with Pull Request branch, don't we need to create issue?

Reply:
The issue only records the current bundle size of the website, so creating an issue isn't essential for the action to run properly.

If you want the bundle size to always be updated through issues, we can keep generating the issues!

What do you think?

Reply:
> If you want the bundle size to always be updated through issues

I don't want to make issue too. Thanks!